ld/nodemcu.ld

```diff
@@ -5,7 +5,7 @@ MEMORY
   dport0_0_seg :                        org = 0x3FF00000, len = 0x10
   dram0_0_seg :                         org = 0x3FFE8000, len = 0x14000
   iram1_0_seg :                         org = 0x40100000, len = 0x8000
-  irom0_0_seg :                         org = 0x40210000, len = 0x80000
+  irom0_0_seg :                         org = 0x40210000, len = 0x90000
```

Comment:
Nope, it fails because i**ram**1_0_seg is too small, not irom1_0_seg. Hence the reference to #1566 which originates from https://github.com/nodemcu/nodemcu-firmware/pull/1435#issuecomment-255995454.